### DISCUSSION TITLE
change color of an image with transparent background

### DISCUSSION THREAD

**Discussion by alexlezam** [dlemstra/Magick.NET#1638] [Help]
In another post I had seen how to change the color in .net with the MagickImage.clut method, but it is still going to be developed.
Is there any other way to change the color of the images? leaving the background transparent

**Comment by dlemstra** [comment]
If the background is fully transparent you can do `image.InverseOpaque(MagickColors.Transparent, MagickColors.Purple)` this will make all non transparent pixels purple.
